mmv1/third_party/terraform/website/docs/r/compute_instance_group_manager.html.markdown

```diff
@@ -221,19 +221,19 @@ update_policy {
 }
 ```
 
-* `minimal_action` - (Required) - Minimal action to be taken on an instance. You can specify either `NONE` to forbid any actions, `REFRESH` to update without stopping instances, `RESTART` to restart existing instances or `REPLACE` to delete and create new instances from the target template. If you specify a `REFRESH`, the Updater will attempt to perform that action only. However, if the Updater determines that the minimal action you specify is not enough to perform the update, it might perform a more disruptive action.
```

Comment:
revert this change since we want to keep the `REFRESH` instead of just REFRESH to provide a nicer view for users looking at the docs.